packages/did-test-utils/src/index.ts

```diff
@@ -0,0 +1,227 @@
+/**
+ * Testing utilites for did, did resolvers, and signing functionality
+ */
+import { CID } from 'multiformats/cid'
+import { DID, DIDResolutionResult, VerifyJWSResult } from 'dids'
+import { wrapDocument } from '@ceramicnetwork/3id-did-resolver'
+import { parse as parseDidUrl } from 'did-resolver'
+import * as KeyDidResolver from 'key-did-resolver'
+import * as sha256 from '@stablelib/sha256'
+import { decode as decodeMultiHash } from 'multiformats/hashes/digest'
+import * as uint8arrays from 'uint8arrays'
+import { CeramicApi } from '@ceramicnetwork/common'
+import { jest } from '@jest/globals'
+
+export const FAKE_CID_1 = CID.parse('bafybeig6xv5nwphfmvcnektpnojts33jqcuam7bmye2pb54adnrtccjlsu')
```

Comment:
all these exported constants seem extremely specific to be exported from a package like this.  This feels like they should belong to the specific tests that are using them.

Reply:
They were actually reused by multiple packages tests, hence consolidating them, rather than duplicating to tests.